index.js

```diff
@@ -28,13 +28,19 @@ function fromVectorTileJs (tile) {
  * Serialized a geojson-vt-created tile to pbf.
  *
  * @param {Object} layers - An object mapping layer names to geojson-vt-created vector tile objects
+ * @param {Object} options - Options to specfic version/extent
+ * @param {Number} options.version - Version of vector-tile spec used
```

Comment:
Since `vt-pbf` won't alter how vector tiles are serialized, I'm not sure if it makes sense to allow this option.

Reply:
The options just a hint, allowing the decoder like 'vector-tile-js' to decode tile correctly.

For example, if one use a extent value 8192 to convert geojson data to tile using 'geojsonvt', then vt-pbf must write this extent, not 4096, to pbf buffer. otherwise decoder will not working. You can see the test case in this PR.

There also has a issues #12 related to this.

Reply:
Sorry for the long delay here.  I see what you mean, and agree that this makes sense.